tox.ini

```diff
@@ -194,6 +195,7 @@ deps =
     jinja29: jinja2>=2.9,<2.10
     jinja210: jinja2>=2.10,<2.11
     memcached: python-memcached
+    molten07: molten>=0.7,<0.8
```

Comment:
As noted in a previous comment, we might want to test all versions of `0.7.x`, or maybe just a pre-0.7.2 and <=0.8?

e.g.:

```
molten071: molten>=0.7,<7.0.2
molten07: molten>=0.7,<0.8
```

What do you think? Just wonder if we have a test case that has a version check if it isn't worth testing the versions between that check?

Or if it _really_ doesn't matter, maybe we remove that check since we'll always install the latest version of `molten` `0.7.x` when we run tests ?